### PR TITLE
[DE] Add `Medals/Unlock_requirements` and its subarticles

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
I've used `von 7.00★ bis 7.99★` instead of `von 7.00★ - 7.99★` for now. The original writing was `7.00★-7.99★`. Not sure if there's a rule for that in German.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Medals/Unlock_requirements/Seasonal_spotlights/de.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+| Summer 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einer der saisonalen Spotlights enthalten ist: Beatmap-Pakete von Sommer 2019 ([osu!](https://osu.ppy.sh/beatmaps/packs/1722)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1723)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1724)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1725)). |
+| Autumn 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einer der saisonalen Spotlights enthalten ist: Beatmap-Pakete von Herbst 2019 ([osu!](https://osu.ppy.sh/beatmaps/packs/1798)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1799)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1800)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1801)). |
+| Winter 2020 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einer der saisonalen Spotlights enthalten ist: Beatmap-Pakete von Winter 2020 ([osu!](https://osu.ppy.sh/beatmaps/packs/1896)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1897)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1898)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1899)). |
```
Comment (respektive) [inline wiki/Medals/Unlock_requirements/Seasonal_spotlights/de.md:11]:
```suggestion
| Summer 2018 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der Beatmap-Spotlights-Pakete von Sommer 2018 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1508)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1509)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1510)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1511)). |
| Fall 2018 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Herbst 2018 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1548)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1549)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1550)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1551)). |
| Winter 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Winter 2019 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1623)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1624)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1625)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1626)). |
| Spring 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Frühling 2019 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1670)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1671)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1672)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1673)). |
| Summer 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Sommer 2019 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1722)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1723)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1724)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1725)). |
| Autumn 2019 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Herbst 2019 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1798)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1799)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1800)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1801)). |
| Winter 2020 Beatmap Spotlights | Schließe alle Beatmaps ab, die in einem der saisonalen Spotlights-Pakete von Winter 2020 enthalten sind ([osu!](https://osu.ppy.sh/beatmaps/packs/1896)/[osu!taiko](https://osu.ppy.sh/beatmaps/packs/1897)/[osu!catch](https://osu.ppy.sh/beatmaps/packs/1898)/[osu!mania](https://osu.ppy.sh/beatmaps/packs/1899)). |
```
I think this is more correct.

File: wiki/Medals/Unlock_requirements/Skill/de.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Freischaltungsanforderungen für Skill Medaillen
+
+| Medaillenname | Anforderung |
+| :-- | :-- |
+| 500 Combo | Erziele eine 500ter Combo auf einer Beatmap in irgendeinem Spielmodus. |
+| 750 Combo | Erziele eine 750ter Combo auf einer Beatmap in irgendeinem Spielmodus. |
+| 1000 Combo | Erziele eine 1000ter Combo auf einer Beatmap in irgendeinem Spielmodus. |
+| 2000 Combo | Erziele eine 2000ter Combo auf einer Beatmap in irgendeinem Spielmodus. |
```
Comment (respektive) [inline wiki/Medals/Unlock_requirements/Skill/de.md:8]:
```suggestion
| 500 Combo | Erziele eine 500er Combo auf einer Beatmap in irgendeinem Spielmodus. |
| 750 Combo | Erziele eine 750er Combo auf einer Beatmap in irgendeinem Spielmodus. |
| 1000 Combo | Erziele eine 1000er Combo auf einer Beatmap in irgendeinem Spielmodus. |
| 2000 Combo | Erziele eine 2000er Combo auf einer Beatmap in irgendeinem Spielmodus. |
```
Not sure but I think you just add `er` instead?

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/Medals/Unlock_requirements/Seasonal_spotlights/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Medals/Unlock_requirements/Seasonal_spotlights/de.md:11]: I agree with you on `enthalten sind`, due to the plural of beatmaps. However, I think my translation is closer to the original: `Clear all beatmaps bundled in one of the Beatmap Spotlights: Summer 2018 beatmap packs`. What do you think about this?

File: wiki/Medals/Unlock_requirements/Skill/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Medals/Unlock_requirements/Skill/de.md]: yeah makes sense. It's `fünfhundert` `er` after all.

File: wiki/Medals/Unlock_requirements/Seasonal_spotlights/de.md (hunk 1 above)
Comment (respektive) [reply wiki/Medals/Unlock_requirements/Seasonal_spotlights/de.md:11]: Hmm yeah, the sentence sounds a bit weirdly worded to me in both languages, but i guess it's fine. updated the suggestion.

**Review by respektive — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**